index.html

```diff
@@ -4,36 +4,12 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/bootstrap/css/bootstrap.css">
     <link rel="stylesheet" href="main.css">
-    <title>My First Website</title>
```

Comment:
Please maintain table on page.

Reply:
We'll add the table later after we have introduced a bunch of other pages.